### PR TITLE
Allowed editing of listing without requiring cover image update

### PR DESCRIPTION
**Pull request by NosBos**
Made it so editing listings can now be done without requiring a cover to be uploaded

### CODE REVIEW

**Review by MillanWang — APPROVED** [review]
Code changes make sense and I tested this to see that the expected functionality works. It seems that we have a linter mismatch across the team which has led to so many tab changes which makes the changes much harder to read. All good though

**Review by MillanWang — CHANGES_REQUESTED** [review]
Change your linter to use tab=4spaces instead of tab=2spaces. This will become an issue on every single file you touch

**Comment by NosBos** [comment]
I thought we were going to use prettier linter? Prettier defaults to 2 space tab

File: frontend/src/components/Seller/SellerListingForm.tsx [hunk 1]
```diff
@@ -6,203 +6,207 @@ import { useHttpClient } from "src/hooks/http-hook";
...
+        errors={errors}
+      />
+      <CustomTextField
+        label="Publisher"
+        name="publisher"
+        control={control}
+        required
+        errors={errors}
+      />
+
```
Comment (GolanHassin) [inline frontend/src/components/Seller/SellerListingForm.tsx]: No need for this spacing

**Review by Justgo13 — APPROVED** [review]

**Review by MillanWang — APPROVED** [review]
Good work. Gotta love lint

**Pull request merged**